interface/core.xhr.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "core.xhr",
+  "api": {
+    // @param {String} method - GET/POST/DELETE
```

Comment:
Nope, we added a compile step that removes them before packaging

On Wed, Jan 7, 2015 at 12:08 PM, Benjamin M. Schwartz <
notifications@github.com> wrote:

> In interface/core.xhr.json
> <https://github.com/freedomjs/freedom/pull/188#discussion-diff-22599227>:
>
> > @@ -0,0 +1,41 @@
> > +{
> > +  "name": "core.xhr",
> > +  "api": {
> > +    // @param {String} method - GET/POST/DELETE
>
> I thought the new JSON parsing didn't accept comments anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/188/files#r22599227>.
>